src/types/utils.ts

```diff
@@ -1,5 +1,13 @@
-export type OptionalExclude<T, Exclusion extends keyof T> = Required<Pick<T, Exclusion>> & Partial<Omit<T, Exclusion>>;
```

Comment:
This utility type, although useful, has been removed to make other types more readable and extractable by `api-extractor`